react-graphql-client-library.md

```diff
@@ -23,31 +23,31 @@ summary = "A tutorial on how to build your own GraphQL client for React applicat
 
 {{% read_before_2 "This tutorial is part 3 of 3 in this series." "Part 1:" "A complete React with GraphQL Tutorial" "https://www.robinwieruch.de/react-with-graphql-tutorial/" "Part 2:" "A complete React with Apollo and GraphQL Tutorial" "https://www.robinwieruch.de/react-graphql-apollo-tutorial/" %}}
 
-In your previous applications, you may have used a GraphQL client library, which is potentially partly view layer agnostic, but can be used with React or other view layer solutions. There are a couple of those GraphQL client libraries out there: Apollo Client, Urql and Relay. In the following, I want to show you that it isn't too difficult to implement your own GraphQL client library that works with React. However, the takeaway shouldn't be "build your own GraphQL client for your production ready applications" but rather:
+You may have used a GraphQL client library that was at least partly view layer agnostic, but still able to work with React or other view layer solution in the form of applications like Apollo Client, Urql and Relay. The next chapter is to illustrate how to implement a GraphQL client library that works with React. The takeaway shouldn't be "build your own GraphQL client for your production ready applications", however, you should instead learn:
```

Comment:
"You may have used a GraphQL client library that was at least partly view layer agnostic, but still able to work with React or other view layer solution in the form of application" to "You may have used a GraphQL client library that was view layer agnostic and thus able to work with React or other solutions such as Angular or Vue. Other GraphQL client libraries (Relay, Urql) aren't so powerful, because they are used only for React."